examples/render-helm-chart-kustomize-inline-values/README.md

```diff
@@ -0,0 +1,66 @@
+# render-helm-chart: Kustomize Inline Values
+
+### Overview
+
+This example demonstrates how to declaratively invoke the `render-helm-chart`
+function with kustomize using the `valuesInline` field.
+
+### Function invocation
+
+To use the function with kustomize, you can specify the `functionConfig`
+in your kustomization's `generators` field. This example uses inline values
+to use instead of the default values accompanying the chart:
+
+kustomization.yaml:
+```yaml
+generators:
+- |-
+  apiVersion: fn.kpt.dev/v1alpha1
+  kind: RenderHelmChart
+  metadata:
+    name: demo
+    annotations:
+      config.kubernetes.io/function: |
+        container:
+          network: true
+          image: gcr.io/kpt-fn/render-helm-chart:unstable
+  helmCharts:
+  - name: ocp-pipeline
+    namespace: mynamespace
+    version: 0.1.16
+    repo: https://bcgov.github.io/helm-charts
+    releaseName: moria
+    valuesInline:
+      releaseNamespace: ""
+      rbac:
+        create: true
+        rules:
+          - apiGroups: [""]
+            verbs: ["*"]
+            resources: ["*"]
+```
+
+Then, to build the kustomization with kustomize v4:
+
+```shell
+kustomize build --enable-alpha-plugins --network .
+```
+
+### Expected result
+
+The following will appear as part of your output:
```

Comment:
Can you please talk a bit more about what does it mean when I see this resource in my output?
How can the user confirm if the inline values file is taking effects? What is the differences when using the default values file vs the inline values file?

You have the following in another example. I think it's good enough if you have something similar
```
You should be able to find `replicas: 5` in
file `deployment_test-helloworld-chart.yaml`, which demonstrates that
the correct values file provided by --valuesFile was used.
```

Reply:
updated the readme